distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc

```diff
@@ -173,23 +173,22 @@ At the bottom of each search is a list of options for the search.
 * *Delete*: Remove this search.
 * *Duplicate*: Create a copy of this search as a starting point.
 * *Search Archived*: Execute this search, but specifically for archived results.
-* *
-Search Historical*: Execute this search, but specifically for historical results.
+* *Search Historical*: Execute this search, but specifically for historical results.
 
 ===== Editing a Search
 
 An existing search can be updated by selecting the search in the *Search* tab of a workspace and by clicking the Edit (image:edit-icon.png[edit icon]) icon.
 
-* *Basic*: Define a <<_text_search_details,Text>>, <<_spatial_search_details,Spatial>>, <<_temporal_search_details,Temporal>>, or <<_type_search_details,Type>> Search.
-
-[[_text_search_details]]Text Search Details:: Searches across all textual data of the targeted data source.
-Text search capabilities include:
-+
-Search for an exact word, such as `Text = apple` : Returns items containing the word "apple" but not "apples". Matching occurs on word boundaries.
-+
-Search for the existence of items containing multiple words, such as `Text = apple orange` : Returns items containing both "apple" and "orange" words. Words can occur anywhere in an item's metadata.
-+
-Search using wildcards, such as `Text = foo*` : Returns items containing words like "food", "fool", etc..
+[no-bullet]
+* *Text*: Perform a minimal textual search that is treated identically to a Basic search with only <<_text_search_details,Text>> specified.
+* *Basic*: Define a <<_text_search_details,Text>>, <<_temporal_search_details,Temporal>>, <<_spatial_search_details,Spatial>>, or <<_type_search_details,Type>> Search.
+[no-bullet]
+** [[_text_search_details]]*Text Search Details*: Searches across all textual data of the targeted data source. Text search capabilities include:
+[no-bullet]
+*** Search for an exact word, such as `Text = apple` : Returns items containing the word "apple" but not "apples". Matching occurs on word boundaries.
+*** Search for the existence of items containing multiple words, such as `Text = apple orange` : Returns items containing both "apple" and "orange" words. Words can occur anywhere in an item's metadata.
+*** Search using wildcards, such as `Text = foo*` : Returns items containing words like "food", "fool", etc..
+*** Text searches are by default case insensitive, but case sensitive searches are an option.
 +
 Wildcards should only be used for single word searches, not for phrases.
```

Comment:
The indentation of these lines seem off. Only the highlighted text should be part of the `Text search capabilities include:` list, right?
![image](https://user-images.githubusercontent.com/8041246/51355072-1b1da680-1a73-11e9-841b-2e5703912f3a.png)